roles/job/templates/job_definition.yml

```diff
@@ -1,3 +1,23 @@
+apiVersion: v1
```

Comment:
@matburt  Hi Matt, could you reivew if the following adding service account and rolebinding to the k8s  runner job definition is fine. Without the rolebinding to link the service account of the k8s runner job  to the ClusterRole tower-resource-operato, The k8s runner job will fail if the k8s job is created outside of the ansible job operator namespace.